lib/outputs/models.py

```diff
@@ -181,9 +183,3 @@ def __unroll_compliance__(self, compliance: list):
             groups = ""
 
         return unrolled_compliance
-
-    def get_csv_header(self):
-        csv_header = []
-        for key in asdict(self):
-            csv_header = csv_header.append(key)
-        return csv_header
```

Comment:
Why do you remove this? Is interesting to have this logic inside that class.

Reply:
I think is no longer used, but yes, please, check it out

Reply:
I removed it because the function was duplicated, it is also in line 58.